com.ktechpit.colorwall.yaml

```diff
@@ -0,0 +1,84 @@
+app-id: com.ktechpit.colorwall
+runtime: org.kde.Platform
+runtime-version: 5.15-22.08
+sdk: org.kde.Sdk
+command: colorwall
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=xdg-download
+  - --filesystem=xdg-pictures
+  - --device=dri
+  - --env=INSTALL_TYPE=flatpak
+  - --talk-name=org.freedesktop.FileManager1
+  - --filesystem=xdg-run/gvfsd
+
+cleanup:
+  - /include
+  - /lib/pkgconfig
+
+modules:
+  - name: lzo
+    config-opts:
+      - --enable-shared
+      - --disable-static
+    cleanup:
+      - /include
+      - /share/doc
+      - '*.la'
+      - /lib/*.so
+    sources:
+      - type: archive
+        url: http://www.oberhumer.com/opensource/lzo/download/lzo-2.10.tar.gz
+        sha256: c0f892943208266f9b6543b3ae308fab6284c5c90e627931446fb49b4221a072
+
+
+  - name: squashfs-tools
+    buildsystem: simple
+    build-commands:
+      - XZ_SUPPORT=1 LZO_SUPPORT=1 make -C squashfs-tools -j ${FLATPAK_BUILDER_N_JOBS} install INSTALL_DIR=${FLATPAK_DEST}/bin
+    cleanup:
+      - '*'
+    sources:
+    - type: git
+      url: https://github.com/plougher/squashfs-tools.git
+      tag: '4.5'
+      commit: '0496d7c3de3e09da37ba492081c86159806ebb07'
+
+
+  - name: libgumbo
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://github.com/google/gumbo-parser/archive/v0.10.1.tar.gz
+        sha256: 28463053d44a5dfbc4b77bcf49c8cee119338ffa636cc17fc3378421d714efad
+    cleanup:
+      - /lib/pkgconfig
+      - /include
+      - '*.a'
+      - '*.la'
+
+  - name: colorwall
+    buildsystem: simple
+    build-commands:
+      - unsquashfs -no-progress tPiGTcpRYwWhmLFqKMefGzuC6isfwbci_9.snap
+      - install -Dm755 squashfs-root/usr/bin/colorwall ${FLATPAK_DEST}/bin/colorwall
+      - mkdir -p ${FLATPAK_DEST}/share/icons/hicolor/
+      - cp -r squashfs-root/usr/share/icons/hicolor/* ${FLATPAK_DEST}/share/icons/hicolor/
+      - install -Dm644 squashfs-root/usr/share/applications/com.ktechpit.colorwall.desktop ${FLATPAK_DEST}/share/applications/com.ktechpit.colorwall.desktop
+      - desktop-file-edit --set-icon=${FLATPAK_ID} ${FLATPAK_DEST}/share/applications/com.ktechpit.colorwall.desktop
```

Comment:
it's surprising you need this as you don't rename any icon.

Reply:
The default desktop file coming from upstream uses a different icon path, thats why :crying_cat_face: 
I will try to align it in next release in upstream and get rid of this line here.

Reply:
removed desktop-file-edit usage.